web/src/muted_topics_ui.js

```diff
@@ -84,41 +100,40 @@ export function mute_topic(stream_id, topic, from_hotkey) {
     });
 }
 
-export function unmute_topic(stream_id, topic) {
-    // Accidentally unmuting a topic isn't as much an issue as accidentally muting
-    // a topic, so we don't show a popup after unmuting.
-    const data = {
-        stream_id,
-        topic,
-        op: "remove",
-    };
-
-    channel.patch({
-        url: "/json/users/me/subscriptions/muted_topics",
-        data,
-        success() {
-            feedback_widget.dismiss();
-        },
-    });
-}
-
 export function toggle_topic_mute(message) {
```

Comment:
In follow-up PR, we should maybe generalize this to, 
`toggle_visibility_policy` or `add_or_remove_visibility_policy`. Namings we can discuss.

(I think, this handles hotkey-toggling, so this should be renamed when working on hotkeys for UNMUTE.)

Reply:
Yes, this can be done when working on hotkeys for UNMUTE as a followup to this PR.